playground/api/v1/api.proto

```diff
@@ -135,6 +135,7 @@ message PrecompiledObject{
   string name = 2;
   string description = 3;
   PrecompiledObjectType type = 4;
+  string pipeline_options = 5;
```

Comment:
Can you document what's the format of the pipeline options? (`option1=val1,option2=val2`? or `--option1=val1 --option2=val2`? Is it the same for Python?)

Reply:
This value is filled using a meta.info file from GCS. So it could be a string with any format.